flash_update/factories/flash_update.py

```diff
@@ -10,7 +10,9 @@
 from flash_update.models import (
     FlashUpdate,
     FlashGraphicMap,
-    FlashAction
+    FlashAction,
+    Donors,
+    DonorGroup
```

Comment:
Let's always add `,` for the last item in the list as well. This creates a better diff for future changes.